### DISCUSSION TITLE
Phi-3.5-MoE

### DISCUSSION THREAD

**Discussion by ayttop** [ggml-org/llama.cpp#9326] [Ideas]
Can it be supported
microsoft/Phi-3.5-MoE-instruct

convert it t5o gguf
؟

**Comment by matiaslin** [comment]
Support for `Phi-MoE` architectures has been recently added (commit  https://github.com/ggerganov/llama.cpp/commit/f8feb4b01af374ad2fce302fd5790529c615710b).
